nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -423,6 +709,14 @@ Nav2Panel::onInitialize()
         navigation_feedback_indicator_->setText(getNavThroughPosesFeedbackLabel());
       }
     });
+
+  // create map_pose subscription for initial_pose
+  map_pose_sub_ = node->create_subscription<geometry_msgs::msg::PoseWithCovarianceStamped>(
+    "map_pose",
```

Comment:
Who's publishing that?

Reply:
Oops my bad, I need to use `amcl_pose`. 

In neo_localization, we have a topic called `map_pose` publishing the pose of the robot on the map. 

Totally forgot about amcl!

Reply:
In fact, SLAM Toolbox also publishes to the AMCL pose topic as well (just so it has drop in replacement functionality with rviz and other tools), you may consider that as well, even if its not "technically" amcl